.pre-commit-config.yaml

```diff
@@ -43,7 +43,7 @@ repos:
       args:
         - -S
 - repo: https://github.com/pycqa/isort
-  rev: 5.10.1
+  rev: 5.12.0
```

Comment:
Without this update, I keep getting an error during commit. [SO link](https://stackoverflow.com/a/75269701)

```
RuntimeError: The Poetry configuration is invalid:
            - [extras.pipfile_deprecated_finder.2] 'pip-shims<=0.3.4' does not match '^[a-zA-Z-_.0-9]+$'
```